### PR TITLE
RN updates v77

### PR DESCRIPTION
**Pull request by js-xc**
see lab issue #6650 for details

### CODE REVIEW

**Comment by js-xc** [comment]
@rschwietzke @jowerner If you have comments/suggestions, let me hear them. Otherwise I'll merge on Friday.

**Comment by rschwietzke** [comment]
Hold of merging till we approve.

File: content/en/xtc/integrations/610-api.md
Comment (rschwietzke) [inline content/en/xtc/integrations/610-api.md:1]: Can you please put the XTC API above Slack?!

**Review by rschwietzke — APPROVED** [review]
Would be nice to have our API listed before Slack in the doc.

**Review by rschwietzke — APPROVED** [review]

**Pull request merged**